keps/993-two-phase-admission/README.md

```diff
@@ -253,6 +254,27 @@ The controller implementing a particular check should:
 * After approving the workload, keep an eye on the check if it starts failing,
   fail the check and cause the workload to move back to the suspended state.
 
+### Preemption Admission Check Controller
+
+Because now, the time when the eviction of the preemption candidate should vary based on the preemptor state
+the scheduler will not issue the eviction during it's process instead it will set a `Pending` admission check
+that is manged by a new built-in admission check controller.
+
+The **Admission Check Controller** will:
+
+- Watch for a change in state of the workloads pending preemption.
+- Watch for workloads that are finishing execution and therefore releasing quota.
+- Watch for AdmissionCheck changes, since the preemption policy can change.
+
+Since evaluating the current preemption state and the needs of a workload requires checking the sate of all resource pool the events are rate limited the kueue's cache is used.
```

Comment:
```suggestion
Since evaluating the current preemption state and the needs of a workload requires checking the state of all resource pool, the events are rate limited and the kueue's cache is used.
```
Not sure if I'm interpreting the sentence correctly.
which events is this referring to?

Reply:
ah, got it now, let me propose another rephrasing:
```suggestion
The preemption controller uses the kueue cache, since it needs to check the state of workloads admitted to the ClusterQueues.
```

Not sure if I'm leaving something out that could be relevant to the design (not sure how the events come into play).